uikit/zoomcontrols/src/main/res/drawable/zoom_controls_button_background.xml

```diff
@@ -1,23 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layer-list xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools">
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:shape="oval">
 
-    <item
-        android:bottom="1.5dp"
-        android:left="1dp"
-        android:right="1dp">
-        <shape android:shape="oval">
-            <solid
-                android:color="?android:attr/colorBackground"
-                tools:color="@color/white" />
+    <solid
+        android:color="?android:attr/colorBackground"
+        tools:color="@color/white" />
 
-            <size
-                android:width="@dimen/zoomControlsButtonSize"
-                android:height="@dimen/zoomControlsButtonSize" />
-            <padding
-                android:bottom="1.5dp"
```

Comment:
naschval si presunul tie paddingy z itemu a odtialto do layout, ci?

Reply:
jj, tady neměli být, to bylo chyba copy/paste z kompasu ;)